lib/Library.php

```diff
@@ -180,6 +182,19 @@ public static function FilterArtwork(string $query = null, string $status = null
 				SELECT *
 				from Artworks', [], 'Artwork');
 		}
+		elseif($status == 'all-submitter' && $submitterUserId !== null){
+			$artworks = Db::Query('
+				SELECT *
+				from Artworks
+				where Status in ("approved", "in_use")
+				or (Status = "unverified" and SubmitterUserId = ?)', [$submitterUserId], 'Artwork');
```

Comment:
Don't bake in `unverified` as as a string, use the enums for this

Reply:
Done. I'll try to clean this up more in the next pass through `Library::FilterArtwork()`.